index.html

```diff
@@ -0,0 +1,301 @@
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="image/yoga.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h2 class="project-title">Tonic</h2>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="separator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="image/art.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="separator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h2 class="title">About Myself</h2>
+          <p class="details">
+            I am a software developer! If you have a project you need coded, do not hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET US CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="image/tw.png" alt="Twitter"
+                /></a>
```

Comment:
- [ ] As the previous reviewer had requested, would you please try to follow the Figma design as much as possible. Some parts of your `<about>` section appear to be displaced. `Hint`  Follow this [link](https://www.w3schools.com/css/css3_flexbox.asp) to learn more about how to use flex boxes. 

![image](https://user-images.githubusercontent.com/98453979/186487448-9c665be4-b5db-4b83-86f4-612bb5304356.png)

### Figma design

![image](https://user-images.githubusercontent.com/98453979/186488390-8e7528fe-1069-41b3-80a7-a5cfb138f12e.png)

Reply:
I did it.